### PR TITLE
Add a button to delete discussions, fix etalab/udata#99

### PR DESCRIPTION
**Pull request by davidbgk**

### CODE REVIEW

**Comment by noirbizarre** [comment]
r+

**Comment by davidbgk** [comment]
@noirbizarre I wonder if we should ask for a confirmation or anything like that, is there something standardized in the admin?

**Comment by noirbizarre** [comment]
Double yes:
- we should ask for confirmation (any deletion)
- there is already some deletion confirmation: https://github.com/etalab/udata-admin/blob/master/js/views/dataset.vue#L151-L156

File: js/components/dataset/delete-modal.vue [hunk 1]
```diff
@@ -10,7 +10,7 @@
             {{ _('You are about to delete this dataset') }}
         </p>
         <p class="lead text-center">
-            {{ _('Are you sure ?') }}
+            {{ _('Are you sure?') }}
```
Comment (noirbizarre) [inline js/components/dataset/delete-modal.vue]: Why removing the space ?

File: js/components/discussions/delete-modal.vue [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+</template>
+
+<script>
+'use strict';
+
+var API = require('api');
+
+module.exports = {
+    components: {
+        'w-modal': require('components/modal.vue')
```
Comment (noirbizarre) [inline js/components/discussions/delete-modal.vue]: With vue 0.12+ it's not needed anymore to have a dash in components name so you can register it as ``modal``

**Comment by noirbizarre** [comment]
r+wc

File: js/components/dataset/delete-modal.vue (hunk 1 above)
Comment (davidbgk) [inline js/components/dataset/delete-modal.vue]: No space in English.

Comment (noirbizarre) [inline js/components/dataset/delete-modal.vue]: OK, my bad

File: js/components/discussions/delete-modal.vue [hunk 3]
```diff
@@ -0,0 +1,55 @@
+<style lang="less"></style>
+
+<template>
+<w-modal title="{{ _('Confirm deletion') }}"
```
Comment (noirbizarre) [inline js/components/discussions/delete-modal.vue]: Be careful, the element name should be the component ie. modal vs w-modal ;)

Comment (davidbgk) [inline js/components/discussions/delete-modal.vue]: Oops, fixed in https://github.com/etalab/udata-admin/commit/004995002f428a187958e40726971435f9816756

**Pull request merged**